ci/install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -ex
+
+main() {
+    curl https://sh.rustup.rs -sSf > rustup-init.sh
+    sh rustup-init.sh --default-toolchain nightly -y -v
+    export PATH="$HOME/.cargo/bin:$PATH"
+    which rustc
+    rustc --version
+    # pip install -r requirements.txt
+    pip freeze
```

Comment:
Guess that's not needed, right?

Reply:
yes, this one is just to make sure what's in the environment.